lib/ssh/doc/src/ssh_client_channel.xml

```diff
@@ -194,7 +194,7 @@
     <func>
       <name since="OTP 21.0">start(SshConnection, ChannelId, ChannelCb, CbInitArgs) -> </name>
       <name since="OTP 21.0">start_link(SshConnection, ChannelId, ChannelCb, CbInitArgs) ->
-      {ok, ChannelRef} | {error, Reason}</name>
+      {ok, ChannelRef} | ignore | {error, Reason}</name>
```

Comment:
I think that the ignore value was intentionally left out as it will not be used by the implementation.

Reply:
I agree